File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/nuri/NUR1017U00GrdNUR1017ListItemInfo.java
package nta.med.data.model.ihis.nuri;

import java.util.Date;

public class NUR1017U00GrdNUR1017ListItemInfo {
	private String infeCode;
	private String bunho;
	private Date startDate;
	private Date endDate;
	private String infeJaeryo;
	private String endSayu;
	private String inputText;
	private String yValue;
	private Double pknur1017;
	private String rowState;

	public NUR1017U00GrdNUR1017ListItemInfo(String infeCode, String bunho,
			Date startDate, Date endDate, String infeJaeryo, String endSayu,
			String inputText, String yValue, Double pknur1017, String rowState) {
		super();
		this.infeCode = infeCode;
		this.bunho = bunho;
		this.startDate = startDate;
		this.endDate = endDate;
		this.infeJaeryo = infeJaeryo;
		this.endSayu = endSayu;
		this.inputText = inputText;
		this.yValue = yValue;
		this.pknur1017 = pknur1017;
		this.rowState = rowState;
	}

	public String getInfeCode() {
		return infeCode;
	}

	public void setInfeCode(String infeCode) {
		this.infeCode = infeCode;
	}

	public String getBunho() {
		return bunho;
	}

	public void setBunho(String bunho) {
		this.bunho = bunho;
	}

	public Date getStartDate() {
		return startDate;
	}

	public void setStartDate(Date startDate) {
		this.startDate = startDate;
	}

	public Date getEndDate() {
		return endDate;
	}

	public void setEndDate(Date endDate) {
		this.endDate = endDate;
	}

	public String getInfeJaeryo() {
		return infeJaeryo;
	}

	public void setInfeJaeryo(String infeJaeryo) {
		this.infeJaeryo = infeJaeryo;
	}

	public String getEndSayu() {
		return endSayu;
	}

	public void setEndSayu(String endSayu) {
		this.endSayu = endSayu;
	}

	public String getInputText() {
		return inputText;
	}

	public void setInputText(String inputText) {
		this.inputText = inputText;
	}

	public String getyValue() {
		return yValue;
	}

	public void setyValue(String yValue) {
		this.yValue = yValue;
	}

	public Double getPknur1017() {
		return pknur1017;
	}

	public void setPknur1017(Double pknur1017) {
		this.pknur1017 = pknur1017;
	}

	public String getRowState() {
		return rowState;
	}

	public void setRowState(String rowState) {
		this.rowState = rowState;
	}

}

File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Nuri/NUR1017U00GrdNUR1017ListItemInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Nuri
{
    [Serializable]
	public class NUR1017U00GrdNUR1017ListItemInfo
	{
		private String _infeCode;
		private String _bunho;
		private String _startDate;
		private String _endDate;
		private String _infeJaeryo;
		private String _endSayu;
		private String _inputText;
		private String _yValue;
		private String _pknur1017;
		private String _rowState;

		public String InfeCode
		{
			get { return this._infeCode; }
			set { this._infeCode = value; }
		}

		public String Bunho
		{
			get { return this._bunho; }
			set { this._bunho = value; }
		}

		public String StartDate
		{
			get { return this._startDate; }
			set { this._startDate = value; }
		}

		public String EndDate
		{
			get { return this._endDate; }
			set { this._endDate = value; }
		}

		public String InfeJaeryo
		{
			get { return this._infeJaeryo; }
			set { this._infeJaeryo = value; }
		}

		public String EndSayu
		{
			get { return this._endSayu; }
			set { this._endSayu = value; }
		}

		public String InputText
		{
			get { return this._inputText; }
			set { this._inputText = value; }
		}

		public String YValue
		{
			get { return this._yValue; }
			set { this._yValue = value; }
		}

		public String Pknur1017
		{
			get { return this._pknur1017; }
			set { this._pknur1017 = value; }
		}

		public String RowState
		{
			get { return this._rowState; }
			set { this._rowState = value; }
		}

		public NUR1017U00GrdNUR1017ListItemInfo() { }

		public NUR1017U00GrdNUR1017ListItemInfo(String infeCode, String bunho, String startDate, String endDate, String infeJaeryo, String endSayu, String inputText, String yValue, String pknur1017, String rowState)
		{
			this._infeCode = infeCode;
			this._bunho = bunho;
			this._startDate = startDate;
			this._endDate = endDate;
			this._infeJaeryo = infeJaeryo;
			this._endSayu = endSayu;
			this._inputText = inputText;
			this._yValue = yValue;
			this._pknur1017 = pknur1017;
			this._rowState = rowState;
		}

	}
}